A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/FMC/A32NX_FMCMainDisplay.js

```diff
@@ -2558,19 +2570,42 @@ class FMCMainDisplay extends BaseAirliners {
         return false;
     }
 
-    setPerfApprTransAlt(s) {
-        if (!/^\d{4,5}$/.test(s)) {
+    trySetPerfApprTransAlt(s) {
```

Comment:
**issue**: `s` doesn't tell me what it is.

Reply:
I just use above code. Copy & paste.

Reply:
It just input fot that function

Reply:
Yes. I can see that.

It's up to developers to make the code easy to understand and read. Please think of a good name 😃.